MAUI/DataForm/localization.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Localization in .NET MAUI DataForm control | Syncfusion
```

Comment:
Data localization in .NET MAUI DataForm control | Syncfusion

Reply:
changed title